### DISCUSSION TITLE
Port MsQuic to PS5

### DISCUSSION THREAD

**Discussion by tldQuantic** [microsoft/msquic#3254] [Q&A]
Hi,

I didn't checked the MsQuic source code yet in detail but I was wondering if you would see some blockers that will prevent me to port MsQuic on PS5?

Thanks for your feedbacks :)

**Comment by nibanks** [comment]
I believe someone might have already done something in this space, so I *think* you should be good to go.

**Reply by tldQuantic** [reply]
Ok, thanks for your feedback.
